main.go

```diff
@@ -53,7 +54,15 @@ func main() {
 			if len(target) == 0 {
 				return errors.New("Missing required argument: TARGET:[TAG]")
 			}
-			return pkg.DoPublish(file, target, digestFile, dryRun)
+			var archShortlist []string
```

Comment:
kind of bikeshedding but "shortlist" may start becoming a loaded word with us. maybe we just call it what it is in this context: `archList`. So maybe we have archListStr and archList variables.

Reply:
ok.

Reply:
updated via forced push